### DISCUSSION TITLE
PTZ question- Keeps alerting even though the area is masked

### DISCUSSION THREAD

**Discussion by balucanb** [VorlonCD/bi-aidetection#270] [Q&A] [answered]
I think I remember there being an answer to this but can't find it. I have a PTZ camera that auto scans back and forth , 4 different positions, I have an area masked off but when the camera swings back past it it still is sending alerts for that area (it is a car in my driveway) the reason this is a problem is I have it all set up to trigger a sound for cars, so hearing BEEP BEEP every few minutes is pretty much a PIA. LOL.  What I would like is to have a custom mask file for each position and for it not to trigger on my car each time it swings past. 

Running the latest beta AI-TOOL version
Thanks

**Comment by VorlonCD** [comment, marked as answer]
This might partially help to avoid so many motion triggers?  https://blueirissoftware.com/forum/viewtopic.php?t=1510

I dont really see a good solution with AITOOL.  You could enable dynamic masking, then go to DETAILS and make each instance of the car from different views "Static"... But this would end up masking out a much larger area than you need and objects you want to be alerted for -might- be ignored.

Or disable the PAN feature or set up 2 cameras to cover the whole area?

**Reply by balucanb** [reply]
Thanks. I fixed it. Got rid of the camera! It was giving me issues anyway. Wanna buy a pyz camera 😂
